plugins/cloud-gce/src/test/java/org/elasticsearch/discovery/gce/GceComputeServiceMock.java

```diff
@@ -91,23 +90,28 @@ private String readGoogleApiJsonResponse(String url) throws IOException {
 
     private String readJsonResponse(String url, String urlRoot) throws IOException {
         // We extract from the url the mock file path we want to use
-        String mockFileName = Strings.replace(url, urlRoot, "") + ".json";
+        String mockFileName = Strings.replace(url, urlRoot, "");
 
         logger.debug("--> read mock file from [{}]", mockFileName);
-        URL resource = GceComputeServiceMock.class.getResource(mockFileName);
+        URL resource = GceComputeServiceMock.class.getResource(mockFileName + ".json");
```

Comment:
I agree with @rjernst here - why do we need to specify a json vs txt here? why not just have a generic `.response` extension where we treat the files as bytes to respond with?